### PR TITLE
add overview table

### PR DESCRIPTION
**Pull request by shannonpileggi**
@jennybc  here is the overview table i mentioned!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,91 +1,129 @@
...
+    wrap: 72
+---
+
 # wtf-workshop
 
 <!-- badges: start -->
+
 <!-- badges: end -->
 
+# 2022 RStudio Conference Overivew
```
Comment (jennybc) [inline README.md]: I feel like this belongs *after* the section that explains the purpose of this repo and that gives an overview of where we keep different types of material.

I don't feel super strongly about this if you do, though.

Comment (jennybc) [inline README.md]:
```suggestion
# 2022 RStudio Conference Overview
```

File: README.md [hunk 2]
```diff
@@ -1,91 +1,129 @@
...
+
+| Content                   | Slides                                                                                                                | Exercises                                                                                                                                                                                                                   |
```
Comment (jennybc) [inline README.md]: Maybe note somewhere that PDFs (or whatever the relevant rendered product is) are often available in workshop repos, but this table is meant to track the underlying source for someone who wants to revise, not just read.

Note that the Google Drive folder isn't world-readable. Folks who need access should ask Jenny.

File: README.md [hunk 3]
```diff
@@ -1,91 +1,129 @@
...
-  1. Central storage for reusable assets that don't belong any where else, such as logos or diagrams.
-  
+1.  A place to record policies about how to run an instance of the "What
+    They Forgot ..." (WTF) workshop, such as what to store where.
+2.  Central storage for reusable assets that don't belong any where
+    else, such as logos or diagrams.
+
 Main types of content and where we store it:
 
-  * [rstats.wtf](https://rstats.wtf) is a bookdown site where we aspire to
```
Comment (jennybc) [inline README.md:13]: Kind of a bummer that the whitespace changes makes this big diff. I don't think anything changed in this section, yeah?

In hindsight, using the visual editor with this file on `main` first and committing the whitespace changes would be a better basis for the PR. But I also don't think it matters that much in this situation.

But if there **are** real changes here, I think it's worth going back and doing this, so we can see those updates.

**Comment by jennybc** [comment]
I'm happy for you to merge this once you decide if/how to take the feedback above. It's definitely a big improvement and I don't want to enforce much back and forth. It's easy to tweak later anyway.

File: README.md (hunk 1 above)
Comment (shannonpileggi) [reply README.md]: done

Comment (shannonpileggi) [reply README.md]: good catch

File: README.md (hunk 3 above)
Comment (shannonpileggi) [reply README.md:13]: Nope, there aren't real changes to the content here. I thought it looked funny, too.

**Pull request merged**